// Copyright 2016 The Chromium Authors
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.
//
// Sync protocol datatype extension for ARC pakcages.

// If you change or add any fields in this file, update proto_visitors.h and
// potentially proto_enum_conversions.{h, cc}.

syntax = "proto2";

option java_multiple_files = true;
option java_package = "org.chromium.components.sync.protocol";

option optimize_for = LITE_RUNTIME;

package sync_pb;

// Properties of arc package objects.
message ArcPackageSpecifics {
  // Unique identifier for the item:
  // Android package name.
  optional string package_name = 1;
  // Android package version.
  optional int32 package_version = 2;
  // Last Android id generated when Android side backups data.
  optional int64 last_backup_android_id = 3;
  // Last time internal value that Android side backups data.
  optional int64 last_backup_time = 4;
}
